root/admin/wikidoc/EditWikiDoc.js

```diff
@@ -8,6 +8,7 @@
  */
 
 import Layout from '../../layout/index.js';
+import {l} from '../../static/scripts/common/i18n/admin.js';
```

Comment:
Any reason not to change the strings to use `l_admin` instead? (I thought it was because these ones are still meant to be extracted/localized, but they seem to be excluded from po/Makefile.)

Reply:
Keeping the code compact is a good point, but particularly for `l_statistics` which is longer and might be horrible to repeat in a file with many strings.

And I guess if we stick with `l`, `ln`, etc., we could still automatically check that they are only imported from `admin.js`.

Reply:
> And I guess if we stick with `l`, `ln`, etc., we could still automatically check that they are only imported from `admin.js`.

I would love to see that, just not too sure how since those are provided for all files through webpack plugin config.

Reply:
One option would be to have script/xgettext.js itself throw an error, since it has the paths of the files it's operating on.

But xgettext.js doesn't see where the functions are imported from. The best we could do there is error if *any* i18n functions are used in these files: so only raw strings would be allowed (and expand2react for the cases where substitution is needed).

Reply:
I expanded CircleCI tests to build a new target `test_no_admin` from `po/Makefile` that will fail if anything is extracted from admin-only files.

Consequently I just dropped this kind of import.